code/api/src/modules/user/resolvers.js

```diff
@@ -68,6 +68,26 @@ export async function getAll() {
   return await models.User.findAll()
 }
 
+// Update user
+export async function update(parentValue, { id, name, email, password, image, address, description }, { auth }) {
+  if(auth.user) {
+    var updatedUser = await models.User.update(
```

Comment:
Glad we got this figured out, I'm sure there is a way we can do this without assigning a variable, but works for now!

Also just a side note I should have mentioned, `var` is pretty unanimously replaced with `const` and `let` for JavaScript now

`const` if the data never needs to be re-assigned, and `let` if it does. In this case `const` would do the trick. 
The reasoning behind this stems from, if I remember correctly, `var` leaks data to the global scope, and can be known to cause issues because of it.

Reply:
yup! ^ also, `const` will need to have an initial value and can never be undefined.

Reply:
Thanks for all the JS knowledge!

Reply:
Thank you frontenders for explaining the difference between `const` and `let` because I was starting to think they are synonymous in JS. The explanation you provided more than cleared it up.